cmd/thanos/compact.go

```diff
@@ -590,9 +642,12 @@ type compactConfig struct {
 	enableVerticalCompaction                       bool
 	dedupFunc                                      string
 	skipBlockWithOutOfOrderChunks                  bool
+	compactionProgressMetrics                      bool
 }
 
 func (cc *compactConfig) registerFlag(cmd extkingpin.FlagClause) {
+	cmd.Flag("progress-metrics", "Enables the progress metrics, indicating the progress of compaction and downsampling").Default("false").BoolVar(&cc.compactionProgressMetrics)
```

Comment:
I mean why even have a flag for this? We get so many questions about how to tell whether Thanos Compact is progressing or not so I think this would be useful for everyone

Reply:
What about still having a flag but default to true? I want users to be able to disable it as some clusters setup doesn't need to check compact backlog

Reply:
Sounds good 👍